### ISSUE TITLE
 onychoheterotopia

### ISSUE THREAD

**Issue by AgneseFeresin** [obophenotype/human-phenotype-ontology#10923]
**Preferred term label:**  onychoheterotopia

**Synonyms** ectopic nail ; supernumerary fingernail 

**Definition (free text, please give PubMed ID)** Onychoheterotopia is a rare condition characterized by the presence and development of a complete nail or part of it outside the ungual plate, generally in the finger. Its pathogenesis is unclear, it may be a congenital anomaly (ectopic germinal cells), part of the polydactyly spectrum, or secondary to trauma (traumatic inoculation of oncocytes). It may be resistant to surgical eradication, or managed through surgical resection.

**Parent term (use hpo.jax.org/app)** Abnormal nail morphology HP:0001597; Abnormal finger morphology HP:0001167

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)** 1 reported patient with Stankiewicz-Isidor syndrome (OMIM #617516) related to PSMD12 (OMIM *604450) PMID: 39641441

**Your nano-attribution (ORCID)** 
https://orcid.org/0000-0002-4187-6282

**Comment by pnrobinson** [comment]
@AgneseFeresin 

Adding new term: 
- ID: HP:6001189
- label: Onychoheterotopia
- parents: [Abnormality of the nail (HP:0001597)](https://hpo.jax.org/app/browse/term/HP:0001597)
- pmid: PMID:23716811|PMID:39641441
- def: Growth of nail tissue outside the ungual plate, generally in the finger.
- comment: 
- orcid: orcid:0000-0002-4187-6282
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10923
- synonyms: Ectopic fingernail(NONE)
